producer-js/web-application/src/createStream.js

```diff
@@ -0,0 +1,6 @@
+const createStream = async function (kinesisvideo, params) {
```

Comment:
Since this is very small function, I don't think it's necessary to create a wrapper and a new file for this.